app/Providers/AclProvider.php

```diff
@@ -12,21 +12,31 @@
 
 namespace Vokuro\Providers;
 
+use Phalcon\Di\DiInterface;
+use Phalcon\Di\ServiceProviderInterface;
 use Vokuro\Application;
 use Vokuro\Plugins\Acl\Acl;
 use function Vokuro\container;
 
-class AclProvider extends AbstractProvider
+class AclProvider implements ServiceProviderInterface
 {
+    /**
+     * @var string
+     */
     protected $providerName = 'acl';
 
-    public function register(): void
+    /**
+     * @param DiInterface $di
+     * @return void
+     */
+    public function register(DiInterface $di): void
     {
         /** @var Application $application */
         $application = container(Application::APPLICATION_PROVIDER);
```

Comment:
@Jeckerson why do we need `container()` in `register(DiInterface $di)`? :)

Reply:
def98b8